app/common/logger.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from telegram import Update
+from telegram.ext import (CallbackContext)
+from config import LOGGING_CHAT_ID
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+
+class Logger:
+
+    def __init__(self, name):
+        self._logger = logging.getLogger(name)
+        self._logger.setLevel(logging.INFO)
+
+    def info(self, message: str):
```

Comment:
I suggest we add the [remaining levels](https://docs.python.org/3/library/logging.html#logging-levels) (WARNING, CRITICAL). would be nice to use them.